monitoring_protected_audience_api_services.md

```diff
@@ -11,59 +15,164 @@ privacy guarantees of such a system, traditional ways of monitoring are not
+| system.memory.usage_kb for main process             | Memory usage                                                                    | Gauge         | Not Noised     | Main Process                                                                   |
+| system.memory.usage_kb for MemAvailable             | Memory usage                                                                    | Gauge         | Not Noised     | MemAvailable                                                                   |
+| system.thread.count                                 | Thread count                                                                    | Gauge         | Not Noised     |                                                                                |
+| initiated_request.count_by_server                   | Total number of requests initiated by the server partitioned by outgoing server | UpDownCounter | Noised with DP | Server name                                                                    |
+| system.key_fetch.failure_count                      | Failure counts for fetching keys with the coordinator                           | Gauge         | Not Noised     | public key dispatch, public key async, private key dispatch, private key async |
+| system.key_fetch.num_keys_parsed_on_recent_fetch    | Number of keys parsed on the most recent key fetch                              | Gauge         | Not Noised     | public key GCP, public key AWS, private key                                    |
+| system.key_fetch.num_keys_cahced_after_recent_fetch | Number of keys currently cached in memory after the most recent key fetch       | Gauge         | Not Noised     | public key GCP, public key AWS, private key                                    |
+
+
+### SFE Metrics
+
+These are metrics tracked on the Seller front end B&A servers.
+
+| Metric                                           | Description                                                                      | Instrument    | Noising        | Attributes      |
+|--------------------------------------------------|----------------------------------------------------------------------------------|---------------|----------------|-----------------|
+| initiated_request.auction.duration_ms            | Total duration request takes to get response back from Auction server            | Histogram     | Noised with DP |                 |
+| initiated_request.auction.errors_count_by_status | Initiated requests by auction that resulted in failure partitioned by Error Code | UpDownCounter | Noised with DP | Absl error code |
+| initiated_request.auction.size_bytes             | Size of the initiated Request to Auction server in Bytes                         | Histogram     | Noised with DP |                 |
+| initiated_request.bfe.errors_count_by_status     | Initiated requests by KV that resulted in failure partitioned by Error Code      | UpDownCounter | Noised with DP | Absl error code |
```

Comment:
The description for this metric should be Initiated requests by "BFE" that resulted.....

Reply:
done.